server/test/repository/ReportingRepositoryTest.java

```diff
@@ -115,9 +121,9 @@ public void loadThisMonthReportingData() {
             // The expected values here have submission duration percentile stats calculated from
             // the submitted (i.e. active and obsolete) applications.
             ApplicationSubmissionsStat.create(
-                "Fake Program A", getMonthTimestamp(today), 3L, 300, 500, 750, 990),
+                "", "Fake Program A", getMonthTimestamp(today), 3L, 300, 500, 750, 990),
             ApplicationSubmissionsStat.create(
-                "Fake Program B", getMonthTimestamp(today), 3L, 300, 500, 750, 990));
+                "", "Fake Program B", getMonthTimestamp(today), 3L, 300, 500, 750, 990));
```

Comment:
suggestion (blocking): update the tests to actually test the change in behavior you've introduced

Reply:
Updated tests.